kubetest/gke.go

```diff
@@ -259,6 +262,18 @@ func (g *gkeDeployer) Up() error {
 			return err
 		}
 	}
+	// Create a custom subnet in that network if it was asked for.
+	if *gkeCustomSubnet != "" {
+		customSubnetFields := strings.Fields(*gkeCustomSubnet)
+		createSubnetCommand := []string{"compute", "networks", "subnets", "create"}
+		createSubnetCommand = append(createSubnetCommand, "--project="+g.network, "--network="+g.network)
```

Comment:
Humm..Should this be `"--project="+g.project`?

Reply:
:joy: you have better eyes

Reply:
Fixed.. thanks.